lib/Dialect/FIRRTL/Transforms/ExtractInstances.cpp

```diff
@@ -424,7 +424,9 @@ void ExtractInstancesPass::extractInstances() {
         inst->getParentOfType<FModuleLike>().moduleNameAttr();
 
   while (!extractionWorklist.empty()) {
-    auto [inst, info] = extractionWorklist.pop_back_val();
+    InstanceOp inst;
+    ExtractionInfo info;
+    std::tie(inst, info) = extractionWorklist.pop_back_val();
```

Comment:
I'm curious why you had to make this change

Reply:
I got this very arcane error when referring to `inst` from within a lamba further down, something like "error: reference to local binding declared in enclosing function". Apparently [because structured bindings aren't variables](https://stackoverflow.com/a/46115028):
> If a lambda-expression [...] captures a structured binding (explicitly or implicitly), the program is ill-formed.

😢

Reply:
Wow, had I had no idea it was different

Reply:
Yeah same here -- it seemed like it was just syntactic sugar for variable declarations. But then again it's C++ and nothing is ever easy.